docs/wallet_API.md

```diff
@@ -728,3 +732,43 @@ In most cases, you want to use the Wallet API when you give the users of your da
 The Wallet API introduces a new method to process operation confirmations. Observables. When the dApp sends the operation to the wallet, the dApp will "listen" when the operation appears in a block. The Wallet API observable will "emit" a stream of events as they occur. Orphan/Uncle block detection is also surfaced when observing confirmations for an operation.
 
 The duties of a Wallet are much broader than the very narrow duty of signing operation byte sequences. That's one reason why we built the Wallet API. It offers a better user experience by doing less. It allows Wallets to carry out all the duties expected of a Wallet.
+
+## Subscribing to events
```

Comment:
subscribing to events is a beacon client only thing, we might want to move this paragraph to above line 80 to give user correct workflow and not mistaken that it might also work for development wallet (temple package)

Reply:
Moved it to a location that is beacon only